src/qibocal/cli/fit.py

```diff
@@ -1,26 +1,53 @@
 import datetime
 import json
+import os
+import shutil
 
 import yaml
 from qibo.backends import GlobalBackend
 from qibolab.serialize import dump_runcard
 
+from qibocal.config import log, raise_error
+
 from ..auto.execute import Executor
 from ..auto.history import add_timings_to_meta
 from ..auto.mode import ExecutionMode
+from ..auto.operation import RESULTSFILE
 from ..auto.runcard import Runcard
 from .utils import META, RUNCARD, UPDATED_PLATFORM
 
 
-def fit(path, update):
+def fit(input_path, update, output_path, force):
     """Post-processing analysis
 
     Arguments:
 
-    - FOLDER: input folder.
-
+    - path: input folder.
+    - update: perform platform update
+    - output_path: new folder with data and fit
     """
-    # load meta
+
+    if output_path is not None:
+        if output_path.exists():
+            if force:
+                # overwrite output_path
+                log.warning(f"Deleting previous directory {output_path}.")
+                shutil.rmtree(os.path.join(os.getcwd(), output_path))
+                path = shutil.copytree(input_path, output_path)
+            else:
+                raise_error(RuntimeError, f"Directory {output_path} already exists.")
+        else:
+            path = shutil.copytree(input_path, output_path)
```

Comment:
This should do the same things with less lines of code:
```suggestion
        if output_path.exists():
            if force is False:
                raise_error(RuntimeError, f"Directory {output_path} already exists.")
            # overwrite output_path
            log.warning(f"Deleting previous directory {output_path}.")
            shutil.rmtree(pathlib.Path.cwd() / output_path)
        path = shutil.copytree(input_path, output_path)
```